content/docs.page.tsx

```diff
@@ -407,7 +407,7 @@ export default function Docs() {
                       'Connect to the XRP Ledger Testnet network to develop and test your apps built on the XRP Ledger, without risking real money or impacting production XRP Ledger users.'
                     )}
                   </p>
-                  <a className="btn btn-primary btn-arrow" href="xrp-testnet-faucet.html">
+                  <a className="btn btn-primary btn-arrow" href="../dev-tools/xrp-faucets">
```

Comment:
Make sure redirects are configured so old links still work after the rename

Reply:
@mDuo13 Said they'd handle that separately :)